shoes-core/lib/shoes/logger.rb

```diff
@@ -1,66 +1,47 @@
 # frozen_string_literal: true
+require 'logger'
+
 class Shoes
-  module Logger
-    class << self
-      def register(name, obj)
-        @loggers ||= {}
-        @loggers[name] = obj
-      end
+  class Logger
+    def initialize
+      @loggers = []
+    end
 
-      def unregister(name)
-        @loggers.delete(name)
-      end
+    def <<(logger)
+      @loggers << logger
+      self
+    end
```

Comment:
this interface tripped me up a bit. It looks a bit weird to do `logger_instance << a_logger` imo as well... it might as well add a log message, you know?

Not saying it needs changing, just a consideration that something like `add_logger` might be good...

But then again, why are loggers added to a logger? Seems weird. Maybe one is the logger and the others are logging_backends? 

Maybe, one is the `LogInterface`/`LogDistributor` and the others are the actual loggers? I hope my naming confusion gets across :D 

Big :+1: for the overall design though, e.g. having one logger thingie with multiple backends :rocket: